ui/app/src/Views/AlertList/Views.elm

```diff
@@ -68,21 +68,38 @@ alertView alert =
 
         b =
             if alert.silenced then
-                buttonLink "fa-deaf" ("#/silences/" ++ id) "blue" Types.Noop
+                buttonLink "fa-deaf" ("#/silences/" ++ id) "blue" Noop
             else
                 buttonLink "fa-exclamation-triangle" "#/silences/new?keep=1" "dark-red" (CreateSilenceFromAlert alert)
+
+        labels =
+            List.filterMap
+                (\( k, v ) ->
```

Comment:
`filterMap` is only needed when you want to filter and map. In this case `filter` is enough.

`labels = List.filter (Tuple.first >> (/=) "alertname") alert.labels`